utils/src/deployer.rs

```diff
@@ -503,6 +564,10 @@ pub async fn deploy_stake_table_proxy(
         .await?;
     let stake_table = StakeTable::new(stake_table_addr, &provider);
 
+    // verify light client and token addresses are contracts
+    assert!(is_contract(&provider, light_client_addr).await?);
```

Comment:
chatted with @sveitser, the design of orchestrator and the question of "should we load init stake table from a static config file" is still open for discussion. 

for now, stop-gap fix is just remove this check, add a TODO that follows https://github.com/EspressoSystems/espresso-network/issues/3163 
@alysiahuggins